src/ADAL.PCL.iOS/AuthenticationContinuationHelper.cs

```diff
@@ -0,0 +1,38 @@
+//----------------------------------------------------------------------
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved
+// Apache License 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------
+
+using System;
+using Foundation;
+
+namespace Microsoft.IdentityModel.Clients.ActiveDirectory
+{
+    public static class AuthenticationContinuationHelper
+    {
+
+        public static bool IsBrokerResponse(string sourceApplication)
+        {
+            return sourceApplication != null && sourceApplication.Equals("com.microsoft.azureauthenticator", StringComparison.OrdinalIgnoreCase);
```

Comment:
This is used when the app receives the response from azure authenticator app. There is no broker version of company portal.